presubmit.py

```diff
@@ -337,7 +304,7 @@ def main() -> int:
 
     args = parser.parse_args()
     os.chdir(_SRC_ROOT)
-    changed_files = get_changed_files()
+    changed_files = [Path(path) for path in diff_utils.get_changed_files()]
```

Comment:
Path(path).absolute() ?

Reply:
Do we need Path(), if we are later converting to str(path).

Reply:
All of this file uses pathlib.Path. The rest of the code generally does not. I can refactor this file but I don't think it is worth it for now, the PR is pretty large as is.